cxx-qt-gen/src/extract.rs

```diff
@@ -517,7 +517,14 @@ fn extract_invokable(
     let return_is_opaque = if let Some(return_type) = &return_type {
         matches!(
```

Comment:
couldn't this use `return_type.qt_type.is_opaque()`?
This feels like code duplication of QtType::is_opaque() :thinking:

Reply:
Done